datafusion/core/src/execution/context.rs

```diff
@@ -1508,6 +1521,40 @@ impl SessionState {
                 )
                 .expect("memory catalog provider can register schema");
 
+            let url = config
+                .config_options
+                .read()
+                .get("datafusion.catalog.location");
+            let format = config.config_options.read().get("datafusion.catalog.type");
```

Comment:
Could you add definitions for these configs in `config.rs` and then run ` cargo run --bin print_config_docs` to generate the markdown for the user guide and copy it into `docs/source/user-guide/configs.md`.